src/main/java/com/restaurante/controllers/CompanyController.java

```diff
@@ -14,22 +20,28 @@ public class CompanyController {
     private final CompanyServices companyServices;
 
     @PostMapping
-    public ResponseEntity<Object> createCompany(@RequestBody CompanyDto companyDto) {
-        return companyServices.createCompany(companyDto);
+    public ResponseEntity<Object> createCompany(@RequestBody CompanyDto companyDto)
+            throws CompanyExistException, InternalServerErrorException {
+        companyServices.createCompany(companyDto);
+        return ResponseEntity.status(HttpStatus.CREATED).body("Company created.");
     }
 
     @GetMapping
     public ResponseEntity<Object> listCompany(){
-        return companyServices.listCompany();
+        List<CompanyDto> lista = companyServices.listCompany();
+        return ResponseEntity.status(HttpStatus.OK).body(lista);
```

Comment:
Acredito que voce pode alterar todos os Object e retornar os objetos corretos no ResponseEntity, como mencionado acima.

Ao inves de usar _HttpStatus.OK_, voce pode usar o proprio metodo do _ResponseEntity_ para isso.
Ele ja ira retornar o **status 200**
Exemplo abaixo
```suggestion
        return ResponseEntity.ok().body(lista);
```